config/locales/reference_request.yml

```diff
@@ -1,7 +1,8 @@
 en:
   reference_request:
-    email:
-      subject: Please give a reference to support the teacher training application of %{candidate_name}
+    subject:
+      initial: Please give a reference to support the teacher training application of %{candidate_name}
```

Comment:
Do we need Please here if we dropped it from the opening heading? Subject is quite long.